text/0000-pub_use_pub_glob.md

```diff
@@ -0,0 +1,122 @@
+- Feature Name: pub_use_pub_glob
+- Start Date: 2025-11-26
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a new syntax for safer glob exports:
+
+```rust
+pub use crate::mymod::pub *;
+```
+
+This re‑exports all items from a module that are already marked `pub`, while excluding private helpers. It balances ergonomics (no need to maintain huge allowlists) with safety (no accidental leakage of private items).
```

Comment:
Isn't this how glob exports already work? In addition to @Diggsey s example, using `pub(super)` isn't exported either.

Do you have example code of something that is exported by globa today that wouldn't be exported with `pub *`?